src/styles.css

```diff
@@ -12,3 +12,6 @@
 .button.white-background:hover { background: white; }
 .button.secondary-background:hover { background: #666; }
 .button.medium-gray-background:hover { background: #999; }
+
+html, body { height: 100%; }
```

Comment:
What are these changes?

Reply:
I guess this is from when I was trying to make the header sticky, I have removed it now because it does not change anything